_includes/search.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  <input id="search-bar" type="text" placeholder="e.g., depth, health, taxes..." />
+  <div id="search-container" />
+</div>
+
+<script src="/assets/js/minisearch.js"></script>
+<script src="/assets/js/search.js"></script>
```

Comment:
Move to `<head>`? For cleaner organization

Reply:
But then they have to be loaded for every page :(